docs/en/observability/create-observability-connectors.asciidoc

```diff
@@ -0,0 +1,105 @@
+[[cases-external-connectors]]
+= Configure external connectors
+
+You can send cases to these third-party systems:
+
+* ServiceNow ITSM
+* ServiceNow SecOps
+* Jira (including Jira Service Desk)
+* IBM Resilient
+* {swimlane}
+
+IMPORTANT: To send cases to external systems, you need the appropriate license, and your role must
+have the *Cases* {kib} privileges as a user. For more details, refer to <<grant-cases-access>>.
```

Comment:
Singular "privilege" or change to "...the appropriate Cases {kib} privileges"?

Reply:
Should be privilege. Thanks.